content/_index.md

```diff
@@ -3,4 +3,36 @@ title = "Stephen Augustus"
 tagline = "Caesar of Systems"
 +++
 
-Friendly neighborhood Kubernetes guy at VMware. Previously Red Hat and CoreOS. Proud geek. Black engineer. Made in NYC.
\ No newline at end of file
+Stephen is a Black engineering director and leader in open source communities.
+
+He currently serves the Kubernetes community as a Chair for
+[SIG Release][sig-release], a Lead for [WG Naming][wg-naming], and an owner of
+[Enhancements][enhancements] subproject.
+
```

Comment:
possible alt: Additionally, he is a [CNCF SIG Contributor Strategy][contrib-strat] Chair and a [Dex][dex] maintainer.

Reply:
Thanks Nicole!
Integrated in 64967387b5640340cc499cd97273717f9e0fffc4.